cnames_active.js

```diff
@@ -1267,6 +1267,7 @@ var cnames_active = {
   "hilo3d": "hiloteam.github.io/Hilo3d",
   "hindenburg": "skeldjs.github.io/Hindenburg",
   "hippy": "zoomchan-cxj.github.io/Hippy",
+  "hiren": "https://hiren2001.github.io/hiren.js.org",
```

Comment:
```suggestion
  "hiren": "hiren2001.github.io/hiren.js.org",
```
As the CI error notes, your entry doesn't need the `https://` prefix

Reply:
@MattIPv4 please check I am Already added This in my mein repository

Reply:
Sorry for The Trouble Now I am Added This In The Repository please Check again